Cargo.toml

```diff
@@ -18,7 +18,7 @@ members = [
 debug = false
 panic = "abort"
 lto = true
-opt-level = 2
+opt-level = 3
```

Comment:
These numbers are very promising, maybe we should switch to using `-O3` for the RTS too. @luc-blaeser did you ever try this?

Reply:
Not yet. I can try this and measure with the GC benchmark...

Reply:
Surprisingly, -O2 is even better than -O3, see https://github.com/dfinity/canister-profiling/pull/62

Reply:
Confirmed from https://github.com/dfinity/canister-profiling/pull/66, the default release profile is already using -O3, the gain here purely comes from LTO.